Examples/outputSinDAC.py

```diff
@@ -1,3 +1,8 @@
+from __future__ import division
+from __future__ import print_function
+from builtins import range
+from builtins import object
+from past.utils import old_div
```

Comment:
The example can be converted to be both Python 2.x and 3.x compatible without these imports from __future__, builtins and past.utils. With my Python 3 updates I've been trying to avoid these.

Also, this example needs to be updated for Windows support as well which I eventually was going to do when I did the Python 3 conversion.